tests/integration/test_versions_in_order.py

```diff
@@ -220,6 +220,7 @@ def prepDiffractionCalibrationIngredients(self, ingredients: Any) -> Any:
 ###############################################################################
 
 
+# @pytest.mark.integration()  # TODO: why is this mark not used?!
```

Comment:
I'm not sure what the mark does, but this test was written to run in the usual unit tests

Reply:
I think eventually we'll need different levels of integration tests, so the `integration` mark could then have a "level" attribute (or something like that).  Right now, the mark allows us to bundle all of the integration tests together, and _not_ run them in the same session that the unit tests are run in.  And, this is not what you want for this test, so you've answered my question, and I'll remove the comments.  :)

Reply:
It might be fine to put this in integration tests, provided we run them as part of the CI/CD.  It is a bit slower than the other unit tests (but still less than 30s), so makes sense to move.
If we get golden data for this test, it could also serve for validation of the diffcal process.